lib/OctokitAction.ts

```diff
@@ -162,7 +163,47 @@ export abstract class OctokitAction extends Action {
     return result;
   }
 
+  protected async sendSlackMessage(text: string): Promise<void> {
+    const channel = this.getInput("slack-channel");
+    if (channel) {
+      this.log("Sending Slack message");
```

Comment:
you should run prettier, since it enforces single quotes

Reply:
> 'prettier' is not recognized as an internal or external command,
> operable program or batch file.

Can you make it run automagically as part of the build?